addons/point_of_sale/static/src/js/models.js

```diff
@@ -121,21 +121,13 @@ exports.PosModel = Backbone.Model.extend({
                 progress: function(prog){ 
                     self.chrome.loading_progress(prog);
                 },
-            }).then(
-                function(){
-                        if(self.config.iface_scan_via_proxy){
-                            self.barcode_reader.connect_to_proxy();
-                        }
-                        done.resolve();
-                },
-                function(statusText, url){
-                        if (statusText == 'error' && window.location.protocol == 'https:') {
-                            var error = {message: 'TLSError', url: url};
-                            self.chrome.loading_error(error);
-                        } else {
-                            done.resolve();
-                        }
-                });
+            }).then(function(){
+                if(self.config.iface_scan_via_proxy){
```

Comment:
What about If you uses this configuration to `False`?
Or another change to disable it since that current runbot.odoo.com for 10.0 is opening the POS correctly:

![screen shot 2019-01-08 at 10 21 46 am](https://user-images.githubusercontent.com/6644187/50843939-4eec2400-132f-11e9-89bf-eb39a619f1ee.png)


FYI I tried reproduce it from the runbot of this customer but there is not a runbot repo with his name.